app/domain/authentication/authn_jwt/signing_key/fetch_cached_signing_key.rb

```diff
@@ -0,0 +1,27 @@
+module Authentication
+  module AuthnJwt
+    module SigningKey
+      # FetchCachedSigningKey is a wrapper of FetchSigningKeyInterface interface,
+      # in order to be able to store the signing key in our cache mechanism. If signing_key_interface don't have
+      # fetch_signing_key it is extreme case that error need to be raised so it can be investigated so reek will ignore
+      # this.
+      # :reek:InstanceVariableAssumption
+      class FetchCachedSigningKey
```

Comment:
is there a reason this is not implemented as a Command Class?

Reply:
You are right